Plugins/Creature/NWScript/nwnx_creature.nss

```diff
@@ -36,6 +36,13 @@ const int NWNX_CREATURE_CLERIC_DOMAIN_TRICKERY    = 19;
 const int NWNX_CREATURE_CLERIC_DOMAIN_WAR         = 20;
 const int NWNX_CREATURE_CLERIC_DOMAIN_WATER       = 21;
 
+const int NWNX_EFFECT_TYPE_ATTACK        = 1;
+const int NWNX_EFFECT_TYPE_DAMAGE        = 2;
+const int NWNX_EFFECT_TYPE_SAVING_THROW  = 3;
+const int NWNX_EFFECT_TYPE_ABILITY       = 4;
```

Comment:
Do you suppose I should rename these as `NWNX_EFFECT_BONUS_TYPE_`?

Reply:
I think `NWNX_CREATURE_BONUS_TYPE_XXX`. So it's properly namespaced and doesn't collide with stuff defined in nwnx_effect.